system/package_pacman.go

```diff
@@ -0,0 +1,55 @@
+package system
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/aelsabbahy/goss/util"
+)
+
+type PacmanPackage struct {
+	name      string
+	versions  []string
+	loaded    bool
+	installed bool
+}
+
+func NewPacmanPackage(name string, system *System, config util.Config) Package {
+	return &PacmanPackage{name: name}
+}
+
+func (p *PacmanPackage) setup() {
+	if p.loaded {
+		return
+	}
+	p.loaded = true
+	// TODO: extract versions
+	cmd := util.NewCommand("pacman", "-Q", "--color", "never", "--noconfirm", p.name)
+	if err := cmd.Run(); err != nil {
+		return
+	}
+	p.installed = true
+	// the output format is "pkgname version\n", so if we split the string on
+	// whitespace, the version is the second item.
+	p.versions = []string{strings.Fields(cmd.Stdout.String())[1]}
```

Comment:
Does pacman support multiple versions of the same package being installed? (ex: multiple kernels)

This code looks like it assumes only 1 package is installed.